### PR TITLE
Remove globing of sources and headers in roofit directory

### PR DESCRIPTION
**Pull request by oshadura**

### CODE REVIEW

**Comment by oshadura** [comment]
@amadio done!

**Comment by amadio** [comment]
Thanks!

**Comment by amadio** [comment]
@egpbos Could you please review these changes? Thank you!

File: roofit/CMakeLists.txt [hunk 1]
```diff
@@ -1,18 +1,3 @@
-if(NOT CMAKE_PROJECT_NAME STREQUAL ROOT)
```
[the remark this answers is not in the document]
Comment (oshadura) [reply roofit/CMakeLists.txt]: Last time it was changed it was 7 years ago...I don't think it is possible due dependencies

**Comment by egpbos** [comment]
On holiday right now, I would be happy to do it when I'm back (Oct 8th). In fact, I think I did something similar in some roofit-dev branch(es). Should really start making some PRs to merge things back into master here... but also after holiday ;)

**Comment by amadio** [comment]
No hurry! Enjoy your holidays!

**Comment by oshadura** [comment]
@phsft-bot build!

**Comment by oshadura** [comment]
Failures are not related to this PR, I so will merge it.

**Pull request merged**